### ISSUE TITLE
Warning: count(): Parameter must be an array or an object that implements Countable

### ISSUE THREAD

**Issue by marlord** [contentful/ContentfulBundle#51]
Hey there,

I get the following errors when I enter any console commands in symfony 5.
In DebugCommand.php line 55:
  Warning: count (): Parameters must be an array or an object that implements Countable

after debug I was able to determine that this is due to the contentful package.
In DebugCommand.php in line 55, "count" is used to check whether the entry of the "client-name" argument should be a mandatory field or an optional field. Php 7.4 doesn't like that :)
This happens if the contentful client is not initialized !!
```php
    # vendor\contentful\contentful-bundle\src\Command\Delivery\DebugCommand.php::__construct (line 55)
        $this->addArgument(
            'client-name',
            \count($this->clients) > 1 ? InputArgument::REQUIRED : InputArgument::OPTIONAL,
            'The name of the client to use'
        );
```
Would be great if you could improve that. A solution could look like this:
```php
        $this->addArgument(
            'client-name',
            !is_null($this->clients) && \count($this->clients) > 1 ? InputArgument::REQUIRED : InputArgument::OPTIONAL,
            'The name of the client to use'
        );
```
Thanks in advance
greetings

Env:
php 7.4.10
symfony 5.2
contentful/contentful-bundle 6.1.0

**Comment by Sebb767** [comment]
Hi marlord,

thanks for reporting this issue - and directly proposing a fix ;)

I'll integrate that in the next release. However, as I'm just taking over the maintenance of this repo at the moment, this might take a bit (I'd estimate ~2 weeks) - I hope this works for you. Thank you!

**Comment by Sebb767** [comment]
Hi @marlord ,

I've just implemented a fix for this. Would you mind checking out the version `dev-issue-51` to check whether the fix works for you before I merge it into master?

Thank you! :)

**Comment by Sebb767** [comment]
This is fixed with version 7.0
